Makefile

```diff
@@ -52,63 +57,76 @@ help:
 	@echo " * 'install' - Install binaries to system locations."
 	@echo " * 'binaries' - Build nerdctl."
 	@echo " * 'clean' - Clean artifacts."
+	@echo " * 'lint' - Run various linters."
 
 nerdctl:
-	$(GO_BUILD) $(GO_BUILD_FLAGS) $(VERBOSE_FLAG) -o $(CURDIR)/_output/nerdctl$(BIN_EXT) $(PACKAGE)/cmd/nerdctl
+	$(GO_BUILD) $(GO_BUILD_FLAGS) $(VERBOSE_FLAG) -o $(CURDIR)/_output/nerdctl$(BIN_EXT) ./cmd/nerdctl
 
 clean:
 	find . -name \*~ -delete
```

Comment:
I am not sure what or why these two are for and if we do need these.
Is that to cleanup leftover files from vi?

Let me know if this should be moved to `find $(MAKEFILE_DIR)`.